src/main/java/org/hiero/sketch/table/PartialMembershipDense.java

```diff
@@ -0,0 +1,103 @@
+package org.hiero.sketch.table;
+
+import org.hiero.sketch.table.api.IMembershipSet;
+import org.hiero.sketch.table.api.IRowIterator;
+import org.scalactic.exceptions.NullArgumentException;
+
+import java.util.*;
+import java.util.function.Predicate;
+
+
+/**
+ * Created by uwieder on 10/20/16.
+ *
+ * This implementation uses a full membership data structure plus a list of filter functions that are used for the isMemeber and for the iterator functions. Upside is that construction is quick, adding a filter function is quick.
```

Comment:
Please use fewer than 100 columns in text. I don't know if the IDE settings are part of the repository or not, but you can set this in the IDE configuration.